packages/clerk-js/src/core/events.ts

```diff
@@ -11,6 +14,9 @@ type TokenUpdatePayload = { token: TokenResource | null };
 
 type EventPayload = {
   [events.TokenUpdate]: TokenUpdatePayload;
+  [events.OrganizationDeleted]: null;
+  [events.OrganizationCreated]: null;
+  [events.UserMembershipDeleted]: null;
```

Comment:
At the moment, I am not interested in passing payload as we are only need to know that an event has fired to trigger the revalidation later.